analytics/analytics/analytic_types/segment.py

```
@@ -0,0 +1,14 @@
+from typing import Optional
+
+class Segment:
+
+    def __init__(self, start_timestamp: int, end_timestamp: int, params: Optional[dict] = None):
+        self.start_timestamp = start_timestamp
```

I think you can raise a error if `start_timestamp` > `end_timestamp`
but stop call it `end_timestamp`, why not to use `to` ?

---

1. `from` is the keyword and can't be used as a field name
2. `_timestamp` suffix makes it less confusing when you have fields with start / from indices also
Raising error is a good idea